.helm/ecamp3/templates/api_cache_deployment.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.apiCache.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "apiCache.name" . }}
+  labels:
+    {{- include "apiCache.selectorLabels" . | nindent 4 }}
+    {{- include "app.commonLabels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "apiCache.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "apiCache.selectorLabels" . | nindent 8 }}
```

Comment:
you could add an annotation with the shasum of the confimap.
Then the deployment is recreated on config changes.
        checksum/config: {{ include (print $.Template.BasePath "/api_cache_vcl_configmap.yaml") . | sha256sum }}
(except if varnish supports hot reloading of the config)

Reply:
Fixed with https://github.com/ecamp/ecamp3/pull/3610/commits/51922783b557ed85d452b1a1957d980477ca8f53

Probably this line is obsolete, because I also included `rollme` (=pod is recreated for each new imageTag/commit). But bether to include both and to be on the safe side.

Reply:
Maybe you only change the config after some problem and then redeploy with the same image tag.
-> checksum/vclConfigmap is needed, the imageTag should anyway be in the image